api-objects/UsersObject.ts

```diff
@@ -1,4 +1,10 @@
-export const authorityTypes: Array<string> = ['Admin', 'Audiologist'];
+export enum AuthorityEnum { 
+	Audiologist, 
+	Statistician, 
+	Admin,
+}
+
+export const authorityTypes: Array<string> = ['Audiologist', 'Statistician', 'Admin'];
```

Comment:
Wait so maybe I'm misreading this, but why have both the strings and the enum?

Reply:
The string is what is held in session storage. The Enumerations correspond to those strings in this array.